### ISSUE TITLE
Fixing limb darkening coefficients

### ISSUE THREAD

**Issue by gapp-c** [sebastian-zieba/PACMAN#19]
I see that there are two limb darkening parameters (_u1_, _u2_) in the fit_par file and the option _fix_ld_ in the pcf. Thus, to fix the LD parameters, one could either fix their values to given numbers in the fit_par file or use model results from exotic-LD for them using the pcf. When I state in both files that I want to fix them, which values have priority?

**Comment by sebastian-zieba** [comment]
Yeah, fixing the limb-darkening (LD) values in PACMAN based on an exotic-ld run hasn't been tested thoroughly yet. If you want to use a literature value for the LD parameter(s) I would just set it in the fit_par. 

I agree that "fix_ld" is a confusing name for a parameter. It should be rather called "use_ld_file" because it should eventually set if the user wants to use the precalculated LD values in the ld_file or not.
